tool/site/client-react/src/components/claim/claimPage.tsx

```diff
@@ -70,10 +73,11 @@ const ClaimPage: React.FC<ClaimPageProps> = ({
       prescriptionDetails.medicationDispenses,
       claimFormValues
     )
-    const response = await axios.post(`${baseUrl}dispense/claim`, claim)
-    const responseStr = JSON.stringify(response.data)
+    const response = await axios.post<ClaimResult>(`${baseUrl}dispense/claim`, claim)
+    console.log(claim)
```

Comment:
we still want these?

Reply:
Until we fix the fact that you don't get errors back from the backend